views/apartments/index.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>App</title>
+</head>
+<body>
+  <h1>Apartments</h1>
+  <% @apartment.each do |apartment| %>
+  <!-- <h2>Delete Apartment</h2> -->
```

Comment:
i think this will cause an error. the variable you instantiated in `app.rb` is `@apartments` not `@apartment`. I think if you fix that one thing, this should work.